app/models/user.rb

```diff
@@ -1,4 +1,11 @@
 class User < ApplicationRecord
+  has_many :follower_follows, foreign_key: 'following_id', class_name: 'Follow'
+  has_many :followers, through: :follower_follows
+
+  has_many :following_follows, foreign_key: 'follower_id', class_name: 'Follow'
+  has_many :followings, through: :following_follows
+  has_many :statuses
+
   validates :handle, length: { in: 4..12 }, format: { with: /\A\w+\z/ }, uniqueness: true
```

Comment:
It seems like it's a good practice to put constant values on a constant variable at the top of the block. You could move your format regex to a constant.

```suggestion
class User < ApplicationRecord
  HANDLE_REGEX = /\A\w+\z/

  has_many :follower_follows, foreign_key: 'following_id', class_name: 'Follow'
  has_many :followers, through: :follower_follows

  has_many :following_follows, foreign_key: 'follower_id', class_name: 'Follow'
  has_many :followings, through: :following_follows
  has_many :statuses

  validates :handle, length: { in: 4..12 }, format: { with: HANDLE_REGEX }, uniqueness: true
```

Reply:
Thanks for the tip!